algos/bitAlgo4/multipleOfThree.java

```diff
@@ -0,0 +1,49 @@
+package bitAlgorithms;
+
+public class multipleOfThree {
```

Comment:
Its a java convention to start a className with Capital Case. Please rename the Class.